schema.ts

```diff
@@ -1383,7 +1383,18 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * TODO(ebo): Unhide this
    * @hidden
    */
-  versionProperty? : PropertyIdentifier<K>;
+  versionProperty?: PropertyIdentifier<K>;
+
+  /**
+   * In case this Object schema is that of a reverse lookup column, store the info to link it back to the source column.
+   * Currently only used for Crossdoc
+   * @hidden
+   */
+  reverseLookupInfo?: {
```

Comment:
The rest of the top-level object schema properties are ones that could apply to any object schema, but this is specific to references. I think an earlier version of this change defined a new interface for the case `type: ValueType.Object` and `codaType: ValueHintType.Reference` which seems correct, this new property should only live on that interface.